packages/cf-layout/src/cf-layout.less

```diff
@@ -45,8 +45,8 @@
 
 @fcm-bg:                        @block__bg;
 
-@fcm-bp-max:                    700px;
-@fcm-bp-min:                    701px;
+@fcm-bp-max:                    600px;
+@fcm-bp-min:                    601px;
```

Comment:
Do we need these any more or could we use the standard breakpoints directly? https://github.com/cfpb/capital-framework/blob/ad6871887c3d851b6dfe34fd6dd36cf63e27c843/packages/cf-core/src/cf-vars-breakpoints.less#L9-L10

Reply:
Talked to Chris to ensure he didn't intend something that I was unaware of. He did not, so I cleaned up the variables.